sources/packages/api/src/utilities/string-builder.ts

```diff
@@ -14,7 +14,7 @@ export class StringBuilder {
    */
   public append(s: string, maxLength?: number) {
     if (maxLength && s.length > maxLength) {
-      s = s.substr(0, maxLength);
+      s = s.substring(0, maxLength);
```

Comment:
These methods have different parameters, changing them here will demand a change in all places where this method is consumed, I believe that something around 51 points. It is very critical for text file generation, e-Certs, and all. Please revert it unless I am missing something.